schema-guide.md

```diff
@@ -313,14 +313,15 @@ Array of [`definitions.reference`](#definitionsreference) objects.
 
 ### `repository`
 
-- type: ...
+- type: [`definitions.url`](#definitionsurl)
```

Comment:
@jspaaks Is this the way we want to reference "simple" definitions (as compared to *objects*)? I think it's the most concise and clear way.

Reply:
Good with me